keras_cv/models/object_detection_3d/center_pillar.py

```diff
@@ -162,29 +58,32 @@ def __init__(
         prediction_decoder,
         **kwargs,
     ):
-        super().__init__(**kwargs)
+        point_xyz = keras.layers.Input((None, 3), name="point_xyz")
+        point_feature = keras.layers.Input((None, 4), name="point_feature")
+        point_mask = keras.layers.Input(
+            (None, 1), name="point_mask", dtype=tf.bool
+        )
+
+        inputs = {
+            "point_xyz": point_xyz,
+            "point_feature": point_feature,
+            "point_mask": point_mask,
+        }
+
+        voxel_feature = voxel_net(
+            point_xyz, point_feature, tf.squeeze(point_mask, axis=-1)
+        )
+        voxel_feature = backbone(voxel_feature)
+        predictions = multiclass_head(voxel_feature)
+
+        super().__init__(inputs=inputs, outputs=predictions, **kwargs)
+
         self._voxelization_layer = voxel_net
-        self._unet_layer = backbone
+        self.backbone = backbone
```

Comment:
maybe prepend with `"_"` to match the others

Reply:
done